### ISSUE TITLE
When changing per page, sorting is reset

### ISSUE THREAD

**Issue by JanPosolda** [ublaboo/datagrid#564]
Hello,
I have problem when I set sorting by any column and then change per page select, sorting is reset and 
don´t remember.
I set sorting by name
![screenshot-ublaboo org 2017-08-18 08-23-48](https://user-images.githubusercontent.com/18554235/29446832-a3f6079e-83ee-11e7-890c-46190951cbcf.png)

then change per page, sorting is reset but arrow show sorting
![screenshot-ublaboo org 2017-08-18 08-24-08](https://user-images.githubusercontent.com/18554235/29446853-c6adc02e-83ee-11e7-81ce-ecbe1837edb9.png)

**Comment by paveljanda** [comment]
Right. You can send a PR, where it would be possible to change that behaviour. (https://github.com/ublaboo/datagrid/blob/v5.x/src/DataGrid.php#L1773)

**Comment by radimvaculik** [comment]
This issue is no longer present in the current version.
